### DISCUSSION TITLE
特定のキーのみが打てなくなることがある

### DISCUSSION THREAD

**Discussion by rinyacho** [Yowkees/keyball#159] [Q&A] [answered]
なぜかDのキー（noballside・Row1・Col3）が時々入力できなくなります🖥
下記が現象となります。

1. 感覚値ですが、1時間に1度くらい一定時間押下入力できなくなります。
2. 10分程すると押下入力できるようになることもあります。
3. 押下入力ができない場合でも、裏側から該当キーのPCBソケットにピンセットをあてて導通すると問題なく入力が行われます。
4. 押下入力ができない場合でも、その他のキーやボール等は正常動作します。
5. 押下入力ができない場合は、OLEDモジュールからも反応が無くなります。
6. 普段はballsideのProMicroを親機としていますが、noballsideを親機としても状況は変わりません。
7. 購入・組立時は該当のキー含め全て正常に動いていました。

下記のことを試してみましたが、改善が見られませんでした。

1. キースイッチの確認と取り替え（ホットスワップ）。
2. ProMicroのRow1及びCol3部分のはんだを確認、再度はんだ付け。
3. 該当キーのPCBソケットの確認と取り替え。
4. 該当キーを担当していると思われるダイオードの確認と取り替え。
5. ProMiroを取り替え（typeCに取り替えたかったので）。

以上となります。
自分で思いつく範囲のことはやってみたのですが、暖簾に腕押しの状態です。。。😇
ご意見いただけると大変助かります ✨
よろしくお願いいたします。  

![moji](https://user-images.githubusercontent.com/97168861/202179079-223029e6-5e49-4fdf-9553-a0868c1773b6.jpg)

**Comment by rinyacho** [comment, marked as answer]
結局のところ、接続に使用しているUSBケーブルを変えたらなおりました。
正常に動いたUSBケーブルと電圧を比較してみましたが、特段差はなく、その他の要素が関係しているようでした。それ以上の検証は機器が無くできていません。
質疑を見ていただいた方、ご心配をおかけしました💦

**Comment by Yowkees** [comment]
ご対応できておらず本当に申し訳ありませんでした。
USBケーブルに関係していたとは驚きですが解決していただきありがとうございました。
よろしくお願いいたします。
